docs/man/rpm-common.8.scd

```diff
@@ -0,0 +1,229 @@
+
+*--load* _FILE_
+	Load an individual macro file.
+
+*--macros* _FILELIST_
+	Replace the list of macro files to be loaded with _FILELIST_.
+	See *Macro Configuration* for details.
+
+*--nodigest*
+	Don't verify package or header digests when reading.
+
+*--nohdrchk*
+	Don't verify database header(s) when retrieved.
+
+*--nosignature*
+	Don't verify package or header signatures when reading.
+
+*--pipe* _CMD_
+	Pipes the output of *rpm* to the command _CMD_.
+
+*--predefine*='_MACRO_ _EXPR_'
+	Defines _MACRO_ with value _EXPR_ before loading macro files.
+	This is a very rare special-purpose switch, use *--define*
+	for the common macro definition needs.
+
+*--quiet*
+	Print as little as possible - normally only error messages will be
+	displayed.
+
+*--rcfile* _FILELIST_
+	Replace the default list of configuration files to be read with
+	_FILELIST_.  See *rpmrc Configuration* for details.
+
+*--root* _DIRECTORY_
+	Use the file system tree rooted at _DIRECTORY_ for all operations.
+	Note that this means the database within _DIRECTORY_ will be used
+	for dependency checks and any scriptlet(s) (e.g. *%post* if
+	installing, or *%prep* if building, a package) will be run after a
+	chroot(2) to _DIRECTORY_.
+
+	Note that rpm assumes the environment inside the root is set up by
+	the caller, such as any mounts needed for the operation inside the
+	root directory.
+
+*--undefine*='_MACRO_'
+	Undefines _MACRO_.
+
+*-v*,
+*--verbose*
+	Print verbose information - normally routine progress messages will
+	be displayed. Supply more than once to increase verbosity, eg. *-vv*.
+
+# DEBUG OPTIONS
+*--debug*
+	Print lots of debug information.
+
+*--fsmdebug*
+	Print debuging information of payload handling code.
+
+*--rpmiodebug*
+	Print debug information about file IO.
+
+*--stats*
+	Print runtime statistics of often used functions.
+
+# DEBUG OPTIONS
+Options which are only useful for troubleshooting.
+
+*--deploops*
+	Print dependency loops as warning.
+
+*--fsmdebug*
+	Print debuging information of payload handling code.
+
+*--rpmiodebug*
+	Print debug information about file IO.
+
+*--stats*
+	Print runtime statistics of often used functions.
+
+# FTP/HTTP OPTIONS
+These options are deprecated. Use special purpose software such as
+*curl*(1) or *wget*(1) for fetching data from the internet.
+
+For some operations, *rpm* can act as an FTP and/or HTTP client so that
+packages can be queried or installed from the internet. Package files for
+install, upgrade, and query operations may be specified as an *ftp* or *http*
+style URL:
+
+_http://HOST[:PORT]/path/to/package.rpm_
+
+_ftp://[USER:PASSWORD]@HOST[:PORT]/path/to/package.rpm_
+
+If both the user and password are omitted, anonymous *ftp* is used.
+
+*rpm* allows the following options to be used with ftp URLs:
+
+*rpm* allows the following options to be used with *http* and *ftp* URLs:
+
+*--httpproxy* _HOST_
+	The host _HOST_ will be used as a proxy server for all *http* and
```

Comment:
Maybe just `The _HOST_ will be used` ?

Reply:
Yeah that's kinda the thing with these obvious placeholders, they easily make explaining it silly :sweat_smile: 
Maybe that should really be "Use _HOST_ as the proxy server for all...", the imperative style is pretty handy for man pages too.

Reply:
Indeed, and this is another thing to note in the Typographic conventions in the template, e.g.
```
- Use imperative style when describing functionality (if possible)
```